blocks/list/block.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "wp-newsletter-builder/list",
+	"version": "0.1.0",
+	"title": "Newsletter List",
+	"category": "text",
+	"icon": "editor-ul",
+	"description": "List wrapper that provides email-friendly enhancements",
+	"textdomain": "list",
+	"editorScript": "file:index.ts",
+	"editorStyle": "file:index.css",
+	"style": [
+		"file:style-index.css"
+	],
+	"render": "file:render.php",
+	"attributes": {
+		"elColor": {
+			"type": "string",
+			"default": "#000"
+		}
+	}
```

Comment:
It seems like the indentation is too large here.

Reply:
I checked this file and `paragraph/block.json`. The indentation in my IDE is correct. I admit that it does look excessive here.